CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+#Contributing to Eclipse Collections
+
+Thanks for your interest in this project.
+
+## Project description:
+
+Eclipse Collections is a collections framework for Java. It has JDK-compatible List, Set and Map implementations with a rich API, additional types not found in the JDK like Bags, Multimaps and set of utility classes that work with any JDK compatible Collections, Arrays, Maps or Strings. The iteration protocol was inspired by the Smalltalk collection framework.
+
```

Comment:
Or did you mean we don't need to have project description in several places? I can get rid of those from CONTRIBUTING.md and Wiki if that's the intention. Please let me know.